src/Sylius/Bundle/ApiBundle/composer.json

```diff
@@ -58,7 +55,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.7-dev"
+            "dev-master": "1.9-dev"
```

Comment:
I believe it should be `1.12-dev` for all packages. We definitely do not update packages' composer.json files regularly :/

Reply:
But I don't know why this puts 1.9....

Reply:
Maybe let's change it manually for all the packages and then try to automatize it with some tool in the refactor workflow?

Reply:
Or not, we still use it on packages themselves 💃 so maybe it should be removed in the main `composer.json` and fixed in those in packages